src/util/refined_string_type.cpp

```diff
@@ -21,10 +21,10 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 #include "std_expr.h"
 
 refined_string_typet::refined_string_typet(
-  const typet &index_type, const typet &char_type)
+  const typet &index_type,
+  const typet &content_type)
```

Comment:
Does `content_type` need to be a `pointer_typet`? If so, perhaps make this only except a `pointer_typet`